Makefile

```diff
@@ -6,4 +6,8 @@ run:
 
 install:
 	pip install -r requirements.txt
+	pip install -r requirements_dev.txt
 
+lint:
+	python -m pylint *.py
```

Comment:
`*.py` will only pick up files in the current directory. fine for now as we have no subdirectories, but something to be aware of that will need to be changed later.

Reply:
I've raised this as a potential issue

Reply:
as a general guide, i'd suggest that if you consider something warrants creating an issue and is also trivial to fix in the work which introduces the issue (i.e. this PR) it makes sense to address it in the current PR.

the alternative would be where, for example, you have had to introduce a workaround due to an upstream bug, which we want to track and fix when possible, but can't do right now.

Reply:
Yeah I did want to try and do it there and then as it seems a quick fix at first, but I was struggling to find a solution, neither ```python -m pylint``` or ```python -m pylint .``` worked. 

As you said on discord this seems to be something else others have encountered: 
https://github.com/PyCQA/pylint/issues/352


Perhaps [this](https://pypi.org/project/pylint_runner/
) may be useful for the future?